Sources/SmileID/Classes/Helpers/LocalStorage.swift

```diff
@@ -199,6 +199,28 @@ public class LocalStorage {
         }
     }
 
+    static func getUnsubmittedJobs() -> [String] {
+        var fileNames: [String] = []
+        do {
+            let files = try fileManager.contentsOfDirectory(atPath: unsubmittedDirectory.relativePath)
+            fileNames.append(contentsOf: files)
+            return fileNames
+        } catch {
+           return fileNames
+        }
```

Comment:
No need for the extra allocation. Let's also add a log for the error case

```suggestion
        do {
            return try fileManager.contentsOfDirectory(atPath: unsubmittedDirectory.relativePath)
        } catch {
           print("Error fetching unsubmitted jobs: \(error.localizedDescription)")
           return []
        }
```

Reply:
fixed